docs/core/install/linux-package-manager-ubuntu.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Install .NET Core on Ubuntu with APT - .NET Core
+description: Install .NET Core SDK and .NET Core runtime on Ubuntu using a package manager.
+author: thraka
+ms.author: adegeo
+ms.date: 05/04/2020
+---
+
+# Install .NET Core with a package manager on Ubuntu
+
+.NET Core is supported on different Ubuntu versions. This article describes how to install .NET Core on Ubuntu with APT. When an Ubuntu version falls out of support, .NET Core is no longer supported with that version. However, these instructions may help you to get .NET Core running on those versions, even though it isn't supported.
+
+## Supported distributions
+
+The following is a list of currently supported .NET Core releases and the versions of Ubuntu they're supported on. These versions remain supported until either the version of [.NET Core reaches end-of-support](https://dotnet.microsoft.com/platform/support/policy/dotnet-core) or the version of [Ubuntu reaches end-of-life](https://wiki.ubuntu.com/Releases).
+
+For the best compatibility, choose a long-term release (LTS) version.
+
+| .NET Core | Ubuntu Version (LTS) | Ubuntu non-LTS Version |
+|-----------|----------------------|------------------------|
+| 2.1 (LTS) | 16.04, 18.04, 20.04  | 19.10                  |
+| 3.1 (LTS) | 16.04, 18.04, 20.04  | 19.10                  |
+
+.NET Core versions no longer supported and the last Ubuntu release it was supported with:
+
+| .NET Core | Ubuntu Version             |
+|-----------|----------------------------|
+| 3.0       | 16.04, 18.04, 18.10, 19.10 |
+| 2.2       | 16.04, 18.04, 18.10        |
+| 2.0       | 14.04, 16.04, 18.04        |
+
+## APT (Advanced Package Tool)
+
+Depending on which version of Ubuntu you're using, these commands are slightly different. The changes are usually to URI paths the command uses. You'll want to modify them to match your version of Ubuntu. These changes are called out with placeholders.
+
+The version of .NET Core in this article is `3.1`. You can replace `3.1` with a different version, such as `2.1` or even an unsupported version like `3.0`. If the command fails, it generally means that version isn't available for your Ubuntu version. If you need an unsupported version, you can try other ways to install such as [Snap](runtime.md#install-with-snap).
+
+The version of Ubuntu used in this article is `19.10`. You can change the feed URIs used here to a supported version such as `16.04`, or `20.04`. You can try to use an unsupported version, such as `17.04` but you're not guaranteed that it will work. If you need an unsupported version, you can try other ways to install such as [Snap](runtime.md#install-with-snap).
```

Comment:
Seems like this will quickly go out of date, and a pain to maintain the versions for every distro in this format.